gp-includes/template-links.php

```diff
@@ -364,18 +364,33 @@ function gp_link_glossary_delete( $glossary, $set, $text = '', $attrs = array()
  * @param WP_User $user A WP_User user object.
  */
 function gp_link_user( $user ) {
+	// This content is escaped in the gp_link_user_get() method.
+	// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+	print( gp_link_user_get( $user ) );
```

Comment:
We don't need to phpcs:ignore:
```suggestion
	echo wp_kses( gp_link_user_get( $user ), array( 'a' => array( 'href' => array(), 'tabindex' => array() ) );
```

Reply:
It is ok, but it needs one more parenthesis at the end.